src/components/search-input/search-input.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import "./search-input.scss";
+import { FaSearch } from "react-icons/fa";
+
+export const SearchInput = ({
+  hasIcon = true,
+  onValueChange,
+  placeholder = "cauta",
+  value
+}) => {
+  const [currentValue, setCurrentValue] = useState(0);
```

Comment:
We should keep the data type the same
```suggestion
  const [currentValue, setCurrentValue] = useState("");
```